### DISCUSSION TITLE
Hot key for "tabbing out" of autocompleted parentheses, quotes, etc

### DISCUSSION THREAD

**Discussion by hamoom** [godotengine/godot-proposals#5248] [Editor]
I find it difficult to type quickly when typing autocompleted parentheses, quotes etc.  It is pretty slow to have to reach and hit the right arrow every time I type one of these characters.  I think godot should come with a configurable hot key to exit those types of auto completed characters (defaults to TAB).  

For reference this is a VSCode extension that accomplishes the same thing.  
https://marketplace.visualstudio.com/items?itemName=albert.TabOut

**Comment by bradlycarpenter** [comment]
You ever find a solution for this?

**Reply by hamoom** [reply]
no. i don't think enough people care about this :(

**Reply by bradlycarpenter** [reply]
Sadge, I guess I will continue to use an external editor then :/

**Reply by Grimshad** [reply]
I also need this

**Comment by fFukurou** [comment]
No way we are almost in 2025 and this is still a thing 💀

**Comment by aXu-AP** [comment]
I think this would be a nice qol addition, I too often find it slowing down typing when I need to move the hand to arrow keys. We would need to keep track which symbols were placed by autocompletion on current line and handle pressing tab in that case.

I think there's no downsides to this as adding tabs in line is quite unusual.
